tests/integration/version.py

```diff
@@ -27,21 +27,21 @@ def run_test(self):
         self.version_expr = re.compile(f".*{ self.options.version }.*")
 
         # check dogecoind with only env
-        dogecoind = self.run_command(["VERSION=1"], [])
```

Comment:
Please revert all changes to this file.

Reply:
Using `self.docker_run` would be fine for you ?
Also, methods `TestRunner.docker_run` and `TestRunner.docker_exec` return directly the output to avoid the use of `.stdout` access & encoding/decoding stuff. Error messages are directly handled on command execution (DockerRunner._shell) and create test failure, not used in tests (at least for now). You would like to restore this also ?